include/zephyr/dt-bindings/pinctrl/it8xxx2-pinctrl-kscan.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2022 ITE Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_IT8XXX2_PINCTRL_KSCAN_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_IT8XXX2_PINCTRL_KSCAN_H_
+
+#define NOT_SUPPORT 0
```

Comment:
?

Reply:
KSI[7:0] pins are in gpio mode, they don't support open-drain and push-pull setting, but KSO[15:0] pins support.
And pinctrlksi node in dtsi will use this define

Reply:
This is namespace pollution in my opinion.

Reply:
Ok, I re-named it.

Reply:
No objections to the new name